shell/edit/logging.banzaicloud.io.output/index.vue

```diff
@@ -24,9 +24,16 @@ export default {
 
   mixins: [CreateEditView],
 
-  data() {
-    const schemas = this.$store.getters['cluster/all'](SCHEMA);
+  async fetch() {
+    await this.$store.dispatch('cluster/findAll', { type: SECRET });
```

Comment:
This shouldn't be needed?

Reply:
Not needed... You are right. Sorry about that. Probably something I added just to do some testing and it stayed on the code. Good catch!